packages/grid/tsconfig.build.json

```diff
@@ -9,6 +9,6 @@
     "types": ["jest", "node"],
     "rootDir": "./"
   },
-  "include": ["./data-grid/src", "./x-grid/src", "./_modules_/**/*"],
+  "include": ["./data-grid/src", "./x-grid/src", "./x-grid-data-generator/src", "./_modules_/**/*"],
```

Comment:
`/x-grid-data-generator/src` don't we want to rename this folder?

Reply:
More appropriate would be 'grid-data-generator' but then there would be a mismatch between the name of the package and the name of the folder.

Reply:
For now, we will keep it as it is and review it at a later stage.

Reply:
The related RFC: https://github.com/mui-org/material-ui-x/issues/226.